tests/sentry/plugins/mail/tests.py

```diff
@@ -213,3 +217,16 @@ def test_notify_users_with_utf8_subject(self):
 
         msg = mail.outbox[0]
         assert msg.subject == u'[Sentry] [foo Bar] ERROR: רונית מגן'
+
+    @mock.patch('sentry.plugins.sentry_mail.models.MailPlugin._send_mail')
+    def test_notify_digest(self, _send_mail):
```

Comment:
This doesn't really test anything right now (not even template rendering) except that the send method was called. Not super useful.